python-package/lightgbm/dask.py

```diff
@@ -238,14 +266,42 @@ def _train(
     -------
     model : lightgbm.LGBMClassifier, lightgbm.LGBMRegressor, or lightgbm.LGBMRanker class
         Returns fitted underlying model.
+
+    Note
+    ----
+
+    This method handles setting up the following network parameters based on information
+    about the Dask cluster referenced by ``client``.
+
+    * ``local_listen_port``: port that each LightGBM worker opens a listening socket on,
+        to accept connections from other workers. This can be differ from LightGBM worker
+        to LightGBM worker, but does not have to.
+    * ``machines``: a list of all machines in the cluster, plus a port to communicate
```

Comment:
This totally just my opinion, but `"machines": a list of all machines in the cluster, plus a port to communicate..."` is kind of circular - the second use of "machines" is referring to IP addresses, but the original use of `machines` (IMO) is an IP:port combo

Reply:
oh yeah I probably should not use the word "machines" again in the description haha, thank you

Reply:
ok updated in https://github.com/microsoft/LightGBM/pull/3994/commits/25462ea00d39cc034fac030415800d615730ae0e, thanks for catching this